deblur/workflow.py

```diff
@@ -438,16 +485,22 @@ def launch_workflow(seqs_fp, working_dir, read_error, mean_error, error_dist,
         filepath(s) to SortMeRNA indexed database for artifact removal
     negate: boolean
         discard all sequences aligning to the ref_fp database
-    threads: integer, optional
-        number of threads to use for SortMeRNA
+    threads_per_sample: integer, optional
+        number of threads to use for SortMeRNA/mafft/vsearch (0 for max available)
     delim: string, optional
         delimiter in FASTA labels to separate sample ID from sequence ID
+    sim_thresh: float, optional
```

Comment:
Same comments as before regarding using None instead of 0.  Also, shouldn't the `coverage_thresh` also be added here?